testing/run_tests.sh

```diff
@@ -0,0 +1,10 @@
+#!bin/bash
+
+set -e
+
+echo "Expect to run from docker, in callek/shipit-test-runner"
+echo "  docker run --rm -v `pwd`:/src -ti callek/shipit-test-runner /src/testing/run_tests.sh"
```

Comment:
I don't think this is the right way to handle running tests in Docker. "shipit-test-runner" won't work as soon as somebody needs to test a patch that adds new dependencies. This seems like the type of container that should be built as needed by everyone. I think the right thing to do here is to add a "docker build" command (and other set-up, such as "npm install" to tox.ini.